server/utilities/constants.js

```diff
@@ -11,7 +11,10 @@ const messages = {
     created : "has been created successfully",
     bad_request : "This is is a bad request",
     not_match : "does not match",
-    not_found : "has not been found"
+    not_found : "has not been found",
```

Comment:
should be indent like  : will be in center

Reply:
Done.